uniffi_bindgen/src/bindings/swift/templates/RustBufferHelper.swift

```diff
@@ -456,3 +485,18 @@ extension Dictionary: ViaFfiUsingByteBuffer, ViaFfi, Serializable where Key == S
         }
     }
 }
+
+{% when Type::Enum with (enum_name) -%}
```

Comment:
@rfk I grabbed the applicable when clauses from the kotlin helper file. Let me know if this looks right.